### ISSUE TITLE
Inconsistent OCR Results with LLaVA 1.6 and Ollama vs. LLaVA Online Demo

### ISSUE THREAD

**Issue by arcaweb-ch** [ollama/ollama#2562]
Hey there, I've posted this issue on [LLaVA repo](https://github.com/haotian-liu/LLaVA/issues/1116) already, not sure if this problem refers to an implementation issue in Ollama. Any idea?

**Comment by easp** [comment]
Are you using the fp16 version? I think the online demo uses an unquantized version of the model.

**Comment by donbr** [comment]
appreciate you posting the issue with both Ollama and LLaVA.

On the Ollama side my concern is that the default model uses Mistral, but the only model supported at higher parameters uses Vicuna.  Refer to the Discord for more info.

The lower parameter model supports both Vicuna and Mistral,.

https://discord.com/channels/1128867683291627614/1128867684130508875/1208258667141402676

**Comment by arcaweb-ch** [comment]
@easp @donbr thanks appreciate your thoughts. I tested both vicuna and mistral versions on different hw setups, they're both producing the same issue. Could be related to a different implementation on Ollama side as stated [here](https://github.com/ggerganov/llama.cpp/pull/5267)?

**Comment by wrapss** [comment]
yes, llava1.6 splits an image into several lower-resolution images for processing, which improves its capabilities. Without this modification (another pr is still pending), the current implementation won't have all the model's capabilities.

**Comment by arcaweb-ch** [comment]
That would actually improve OCR operations a lot (trying to catch @jmorganca attention :)

**Comment by donbr** [comment]
Apologize for missing the updates.  Recommend that we work with the llava team to improve our test scenarios.  I posted a discussion item in their GitHub that is related.

**Comment by donbr** [comment]
@arcaweb-ch did you receive an answer from @jmorganca on this?  What does Ollama currently have in the form of regression tests for LLaVA?

My test case was comparing Image Analysis abilities across LLaVA / OpenAI / Gemini, and their ability to tell the difference between a Werewolf and a Wolf.  LLaVA 1.5 on Ollama performed consistently better than the others until 1.6.

- [Discussion on LLaVA site](https://github.com/haotian-liu/LLaVA/discussions/1157)
- [AI Vision Image Analysis / Classification Using Ollama](https://github.com/donbr/visionary_storytelling/blob/main/notebooks/ai_vision_image_classification_ollama.ipynb) - a Jupyter notebook using Ollama LLaVA and Dolphin-Mistral.

**Comment by deviswot** [comment]
Can some one please explain why LLaVA v1.5 has two models on modelfile while new v1.6 has only 1 model on modelfile?

**Comment by ChristianWeyer** [comment]
> yes, llava1.6 splits an image into several lower-resolution images for processing, which improves its capabilities. Without this modification (another pr is still pending), the current implementation won't have all the model's performance.

Has this all been integrated already?

I am running latest Ollama 0.1.38 and still see this: https://github.com/haotian-liu/LLaVA/issues/1497#issuecomment-2117167208
Thanks!
